src/parser/core/modules/racials/dwarf/Stoneform.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Analyzer from 'parser/core/Analyzer';
+
+import Combatants from 'parser/core/modules/Combatants';
+import { formatNumber } from 'common/format';
+import RACES from 'game/RACES';
+import SpellIcon from 'common/SpellIcon';
+import SPELLS from 'common/SPELLS';
+import StatisticBox from 'interface/others/StatisticBox';
+
+/**
+ * Removes all poison, disease, curse, magic, and bleed effects and reduces all physical damage taken by 10% for 8 sec.
+ *
+ * Example log: https://www.warcraftlogs.com/reports/gYCR2kZaJHVbjfxP/#fight=52&source=16&type=damage-taken
+ */
+
+const STONEFORM_DAMAGE_REDUCTION = 0.1;
+const FALLING_DAMAGE_ABILITY_ID = 3;
+const PHYSICAL_EVENT_TYPE = 1;
```

Comment:
Please use the `MAGIC_SCHOOLS.ids.PHYSICAL` for the Physical event type id, rather than listing it yourself. You can import `MAGIC_SCHOOLS` from `common/MAGIC_SCHOOLS`.